cmd/jag/commands/util.go

```diff
@@ -80,6 +80,10 @@ func (s *SDK) VersionPath() string {
 	return filepath.Join(s.Path, "VERSION")
 }
 
+func (s *SDK) DownloaderPath() string {
```

Comment:
Not really happy about the name "DownloaderPath".
To me that's the tool that dowloads something.
Maybe `DownloadInfoPath` ? I would also add a comment here what that file contains.

Reply:
Changed to `DownloaderInfoPath`.